src/components/Todo/Todo.tsx

```diff
@@ -0,0 +1,70 @@
+import classNames from 'classnames';
+import { useEffect, useState } from 'react';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+  spinner: boolean;
+  onDeleteTodo: (todoId: number) => void;
+  isDeleting: boolean;
+  deletingCompleted: boolean;
+};
+
+const TodoComponent: React.FC<Props> = ({
+  todo, spinner, onDeleteTodo, isDeleting, deletingCompleted,
+}) => {
+  const [hasSpinner, setHasSpinner] = useState(spinner);
+
+  const isActive = todo.completed && deletingCompleted ? true : hasSpinner;
+
+  useEffect(() => {
+    if (!isDeleting) {
+      setHasSpinner(false);
+    }
+  }, [isDeleting]);
+
+  const deleteHandler = (todoId:number) => {
+    onDeleteTodo(todoId);
+    setHasSpinner(true);
+  };
+
+  return (
+    <div
+      data-cy="Todo"
+      className={
+        classNames('todo', {
+          completed: todo.completed,
+        })
+      }
```

Comment:
```suggestion
      className={classNames('todo', {
        completed: todo.completed,
      })}
```

So you will have more compact syntax